crates/types/src/field_dimensions.rs

```diff
@@ -35,4 +35,10 @@ impl FieldDimensions {
         position.x().abs() > self.length / 2.0 - self.goal_box_area_length
             && position.y().abs() < self.goal_box_area_width / 2.0
     }
+
+    pub fn is_inside_kick_off_target_region(&self, position: Point2<Field>) -> bool {
+        position.x().signum() == position.y().signum()
+            && position.x().abs() < self.width / 2.0
+            && position.x().abs() <= position.y().abs()
+    }
 }
```

Comment:
Im not sure this should be a method on `FieldDimensions` since its implementation seems very specific for the kickoff behavior. Maybe make this a free function instead

Reply:
Done